### DISCUSSION TITLE
mpi4py for multiple MPI stacks

### DISCUSSION THREAD

**Discussion by drmichaelt7777** [mpi4py/mpi4py#100] [Ideas]
Can you suggest a way to install different mpi4py instances one for each different MPI stack? It is common to use say openmpi for some codes and intelmpi or mpich for others. 

thanks

**Comment by dalcinl** [comment]
* Different conda environments.
* Different virtual environments.
* mpi4py installed in two different directories and use `PYTHONPATH`.
* mpi4py installed in two different directories and use `*.pth` files.
